packages/core/src/shared/ui/wizardPrompter.ts

```diff
@@ -51,6 +61,15 @@ export class WizardPrompter<T> extends Prompter<T> {
 
     protected async promptUser(): Promise<PromptResult<T>> {
         this.response = await this.wizard.run()
-        return this.response
+
+        if (this.response === undefined) {
+            return WIZARD_BACK as PromptResult<T>
+        } else if (_.isEmpty(this.response)) {
```

Comment:
is lodash actually needed here? we are trying to eliminate it

Reply:
This is not a necessity.

Reply:
Updated.